catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -86,9 +85,17 @@
     public void init() {
         try {
             Security.runAsAdminWithException(() -> {
-                Optional<Metacard> optional = federationAdminService.getLocalRegistryIdentityMetacard();
+                Optional<Metacard> optional =
+                        federationAdminService.getLocalRegistryIdentityMetacard();
                 optional.ifPresent(e -> System.setProperty(RegistryConstants.REGISTRY_ID_PROPERTY,
                         RegistryUtility.getRegistryId(e)));
+                if (optional.isPresent()) {
```

Comment:
Seems like this section and the `optional.ifPresent` right above could be combined.